config/config-sil.yaml

```diff
@@ -1,9 +1,11 @@
+settings:
+  telemetry_enabled: true
```

Comment:
Propably we want to enable this in the other configs as well

Reply:
Not sure if we need it everywhere. maybe one example on how to turn it on is enough